src/main/webapp/app/routes/routes.tsx

```diff
@@ -44,6 +45,7 @@ const Routes: React.FunctionComponent<IRoutesProps> = (props: IRoutesProps) => {
             <PrivateRoute exact path={PAGE_ROUTE.SEARCH} component={SearchPage} />
             <PrivateRoute path={PAGE_ROUTE.CURATION} component={CurationRoutes} hasAnyAuthorities={[AUTHORITIES.CURATOR]} />
             <PrivateRoute exact path={PAGE_ROUTE.ACCOUNT} component={Account} hasAnyAuthorities={[AUTHORITIES.USER]} />
+            <PrivateRoute exact path={PAGE_ROUTE.VARIANR_REC} component={VariantRecListPage} />
```

Comment:
Let's add `hasAnyAuthorities={[AUTHORITIES.CURATOR]}`.

Could you also add authorization to the backend too?